blog/_posts/2020-02-12-local-dev.md

```diff
@@ -0,0 +1,86 @@
+---
+slug: local-dev
+date: 2020-02-11
+author: dan
+layout: blog
+title: "Non-Kubernetes Local Dev"
+subtitle: "Using Tilt to replace your start.sh"
+image: /assets/images/local-dev/uncontained.webp
+image_caption: "The goose from Untitled Goose Game feeling the freedom of not being in a container anymore."
+tags:
+  - tilt
+  - local
+keywords:
+  - tilt
+  - local
+---
+
+Many teams that use Kubernetes for production don't use Kubernetes in development to avoid Kubernetes pain points (like the slow inner-loops of building container images). But these teams still feel the pains of multi-service development:
+* Updating the right subset of microservices after you edit files is manual/error-prone/ad hoc even if you're just running bare processes on your laptop.
```

Comment:
It feels weird that this first bullet doesn't have a question in it, but the other two do. I think they should be consistent: either all be questions or statements of problems.